### ISSUE TITLE
Have a bot ping closed PRs when they arrive in core-sdk builds

### ISSUE THREAD

**Issue by anurse** [dotnet/aspnetcore#9299]
In the same vein, given repo X and commit C (where C is the latest commit in core-sdk), have a bot post a comment to all merged PRs that are contained in C and were closed since the last time the bot ran (to avoid pinging eons-old PRs 😉) and ping the author in a comment:

1. Sample content: `@[author] this change is now available in dotnet/core-sdk version [version]. You can download it at [URL]`
1. The URL should go to a page that has all assorted download links, specifically for “[version]” (though as a first pass, it would be OK to just link to the dotnet/core-sdk README with the latest links)
1. If people think this might be spammy, it could be opt-in based on GitHub author/team membership/etc. My team said they’d like it quite a lot and would not be annoyed by the noise. I can’t speak for other engineers though. 
1. This should be doable now with Maestro’s API, or even just by scraping the latest.version file in the blob storage (I’ve done it manually a number of times).
1. I can probably lend a resource (post-preview5 branch closure) to this effort to get it done, as long as the Build Team is able to review/maintain it and provide guidance. I’d love to have this in place early in the preview6 milestone (which is where we’re looking to ramp up “routine” validation efforts).

**Comment by anurse** [comment]
This I would like us to drive as a team rather than waiting on "common features" in arcade. I do think we could contribute this forward to the arcade ecosystem though!

**Comment by natemcmaster** [comment]
I think ownership of this feature lives with the dnceng team. Let's leave open until our next Arcade sync up to ask about 

1. if there are plans for someone on dnceng to implement this
2. and if not, would they accept contributions to the bot from our team to make this possible.

Until then, putting this in build team backlog.

**Comment by markwilkie** [comment]
Interesting idea.  Thoughts @mmitche ?

**Comment by mmitche** [comment]
Yeah, interesting, and fits along with another epic that I was looking to do at some point (tracking merged PRs through the system).

Knowing the delta on both sides of the subscription is pretty interesting.  For instance, both:
- A set of target repos you want to watch for when the update is merged in those repos
- Links to "contained updates" in a PR.  So we may know that extensions package X is being updated in aspnetcore-tooling, but we don't highlight today that that update also carries N other updates from repos further upstream.

**Comment by anurse** [comment]
> tracking merged PRs through the system

I know many people on the team who would **love** to be able to click a button on a particular build of repo X and see all the PRs merged into repo X **and it's dependencies** since the last build.

The "ping" behavior is also very useful for the feature verification work we want to try and enable. I'd like engineers to be able to see this ping, install the referenced (or just the latest) build and then verify that their feature is present and functioning.

As I say in the original post, this is very important to us and we'd be up to help make it happen. Whenever I talk to engineers about verification the *first* concern is "but how do I know my change is in the final build?".

**Comment by mmitche** [comment]
Okay thanks.  We'll need to come up with a design (UX + actual implementation).  I don't know how soon this could be done, as there is a rather large lineup of other things that need to be done for P7 that take higher precedence.

**Comment by mmitche** [comment]
Another thought I had was that this should be a generalized app that produces a check on a PR

**Comment by Pilchie** [comment]
See https://github.com/dotnet/arcade/issues/2518.  @mmitche says this is very hard.  Probably not something we should try to take on on our own.
